Drop-In/src/main/java/com/braintreepayments/api/DropInClient.java

```diff
@@ -66,7 +84,19 @@ public DropInClient(Context context, String authorization, DropInRequest dropInR
     }
 
     DropInClient(Context context, String authorization, String sessionId, DropInRequest dropInRequest) {
-        this(createDefaultParams(context, authorization, dropInRequest, sessionId));
+        this(createDefaultParams(context, authorization, null, dropInRequest, sessionId, null, null));
+    }
+
+    public DropInClient(FragmentActivity activity, DropInRequest dropInRequest, ClientTokenProvider clientTokenProvider) {
```

Comment:
I see we don't have doc strings on the original public constructor, but might be helpful to add them for all of these to clarify when to use each.

Reply:
Yeah for sure maybe in a separate PR before release? That might be a good task to do in tandem with the docs site PR.